contracts/src/CrossChainApplications/examples/NativeTokenBridge/ERC20TokenSource.sol

```diff
@@ -30,10 +30,9 @@ contract ERC20TokenSource is
     ITokenSource,
     ReentrancyGuard
 {
-    // The address where the burned transaction fees are credited.
-    // Defined as BLACKHOLE_ADDRESS at
-    // https://github.com/ava-labs/subnet-evm/blob/e23ab058d039ff9c8469c89b139d21d52c4bd283/constants/constants.go
-    address public constant BURNED_TX_FEES_ADDRESS = 0x0100000000000000000000000000000000000000;
+    // Designated Blackhole Address for this contract. Tokens are sent here to be "burned" when
+    // a SourceAction.Burn message is received from the destination chain.
+    address public constant BURN_ADDRESS = 0x0100000000000000000000000000000000010203;
```

Comment:
Not for this PR, but are there lower hanging clean up options here to create `TokenSource` parent contract that both `ERC20TokenSource` `NativeTokenSource` could inherit from?

That way we wouldn't have to define this address in multiple places, and have an abstract `_burnTokens` method, etc.

Reply:
Yeah, we could share about half the code, so I think it would be worthwhile. I'll make a ticket.